jest-setup.ts

```diff
@@ -7,6 +7,7 @@ if ( typeof window !== 'undefined' ) {
 }
 
 nock.disableNetConnect();
+nock.enableNetConnect( 'raw.githubusercontent.com' );
```

Comment:
I had to enable the requests to download wp-cli.phar for testing
https://github.com/Automattic/studio/pull/203/commits/5d61bc1b11c5ffe523bdf091d2115beb5d79f3ac

The error I was receiving in https://buildkite.com/automattic/studio/builds/1005#018ff2d2-fad8-490e-a880-b77ed0b45d59 is:
```
 Unhandled error. (NetConnectNotAllowedError {
   name: 'NetConnectNotAllowedError',
   code: 'ENETUNREACH',
   message: 'Nock: Disallowed net connect for "raw.githubusercontent.com:443/wp-cli/builds/gh-pages/phar/wp-cli.phar"'
 })
```